tests/CMakeLists.txt

```diff
@@ -195,6 +195,18 @@ target_link_libraries(gsl_tests
 )
 add_test(gsl_tests gsl_tests)
 
+# Ignore uninteresting warning about ABI change somewhere inside std::regex. GSL never supported GCC 5.
+# The warning was removed in GCC 12, see: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=93011
+if (CMAKE_SYSTEM_PROCESSOR STREQUAL "ppc64le"  # PowerPC 64-bit Little-Endian
```

Comment:
would be it be possible to merge this logic up into where `gsl_tests_config` is defined initially? Perhaps not if you're conditionally switching between `set_property` and `target_compile_options` (see my other comment).